scene/realistic-lighting-and-shadows/src/main/resources/realistic_lighting_and_shadows/main.fxml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 Esri.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+<?import javafx.scene.layout.*?>
+
+<?import com.esri.arcgisruntime.mapping.view.SceneView?>
+<?import com.esri.arcgisruntime.mapping.view.LightingMode?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.collections.FXCollections?>
+<StackPane fx:controller="com.esri.samples.realistic_lighting_and_shadows.RealisticLightingAndShadowsController"
+           xmlns="http://javafx.com/javafx"
+           xmlns:fx="http://javafx.com/fxml" stylesheets="/realistic_lighting_and_shadows/style.css">
+    <!--SDK SceneView-->
+    <SceneView fx:id="sceneView"/>
+    <StackPane.margin>
+        <Insets topRightBottomLeft="10" />
+    </StackPane.margin>
+    <!--Slider box-->
+    <VBox styleClass="panel-region">
+        <Label fx:id="time"/>
+        <ComboBox fx:id="comboBox" maxWidth="Infinity">
+            <items>
+                <FXCollections fx:factory="observableArrayList">
+                    <LightingMode fx:value="NO_LIGHT"/>
+                    <LightingMode fx:value="LIGHT"/>
+                    <LightingMode fx:value="LIGHT_AND_SHADOWS"/>
+                </FXCollections>
+            </items>
+        </ComboBox>
+        <Slider fx:id="timeSlider" value="12" max="24" majorTickUnit="4" minorTickCount="60" showTickMarks="true"
```

Comment:
If we multiply the min/max values here by 60, we can get the values from the slider represented in minutes. It might make the calculation in `changeTimeOfDay` a tad simpler by simply doing
```
int hours = sliderValue / 60;
int minutes = sliderValue % 60;
```

I accept that this is a matter of preference though, so up to you :)